crates/rpc/rpc-types/src/eth/pubsub.rs

```diff
@@ -132,6 +132,10 @@ impl<'a> Deserialize<'a> for Params {
             return Ok(Params::None)
         }
 
+        if let Some(val) = v.as_bool() {
```

Comment:
Thanks @mattsse, just for my sanity could we not have done this

```
    if let Some(val) = v.as_bool() {
       return Ok(Params::NewPendingTransactions(val))
    }
```

In essence ensure the bare bool val is wrapped in the enum variant